### ISSUE TITLE
'All Stories' post images are very small in mobile view

### ISSUE THREAD

**Issue by piyushvarshney** [wowthemesnet/mundana-theme-jekyll#47]
Hi @wowthemesnet , love your work!
But there's an issue what I think is not apt. When we make the site go to mobile mode, the "All Stories" section post images are very small, this should be fixed or either the images should be made large and come  above the post title in responsive mode. Is that possible? 
![Screen Shot 2020-05-07 at 00 23 29](https://user-images.githubusercontent.com/14103096/81216814-2b0b2200-8ff9-11ea-87e7-d1999bdd198d.png)

**Comment by wowthemesnet** [comment]
You're right. It was something I wanted to do anyway . I'll push an update soon.

**Comment by piyushvarshney** [comment]
I did it for web version by tweaking the CSS class. Not sure about the mobile-view though. Please let me also know if there's a direct way out. @wowthemesnet

**Comment by piyushvarshney** [comment]
@wowthemesnet Any updates?
